torch/optim/lr_scheduler.py

```diff
@@ -77,7 +93,13 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
+    def get_last_lr(self):
```

Comment:
I'm ok with having it being a method called `get_last_lr`, but I wonder if it was a `@property` called `lr` it wouldn't be more natural to users? Like
```python
print(i, scheduler.lr)
```

Reply:
The reason I went for `get_last_lr` was to also emphasize that `get_lr` does not return what the user would expect from it, the last computed lr. If there's an `lr` property, then it looks like `get_lr` should return the learning rate.

I originally wanted to change the name of `get_lr`, but that led to many BC breaking change internally, since `get_lr` is used to compute the new learning rate in `step`.

I also wanted to change the names to "values" to remove the reference to "lr" so as to be more general later (so that we can have weight schedulers with identical syntax). I'll leave that for a separate PR.

Thoughts?